zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementService.kt

```diff
@@ -56,4 +65,27 @@ class ObjectManagementService(
     fun getAll(): MutableList<ObjectManagement> = objectManagementRepository.findAll()
 
     fun deleteById(id: UUID) = objectManagementRepository.deleteById(id)
+
+    fun getObjects(id: UUID, pageable: Pageable): PageImpl<ObjectWrapper> {
+        val objectManagement = getById(id)!!
```

Comment:
Try to not use `!!` but instead do a null check that shown a helpful log message so that when this occurs it's easy to debug.